kubespawner/spawner.py

```diff
@@ -433,6 +442,15 @@ def _hub_connect_port_default(self):
         """
     )
 
+    user_storage_match_labels = Dict(
+        help="""
```

Comment:
ok! I think  we should set the default explicitly, and explain why you should use this and when in the help. The current help text isn't sufficient, IMO.

Also in the future, it'd be nice to have these as a separate PR, rather than as part of one :) Makes it easier to review.

I'll do some testing and try to get this merged tomorrow!